solutions/tenants/outputs.tf

```diff
@@ -0,0 +1,6 @@
+##############################################################################
+# Outputs
+##############################################################################
+
+
```

Comment:
We are going to need to add an output here. It will have to be a list of maps output since our input supports creating multiple tenants. Each map should contain the tenant name, id and crn